modules/data.remote/tests/Rcheck_reference.log

```diff
@@ -80,49 +33,24 @@ Author field differs from that derived from Authors@R
 * checking whether the namespace can be loaded with stated dependencies ... OK
 * checking whether the namespace can be unloaded cleanly ... OK
 * checking loading without being on the library search path ... OK
-* checking use of S3 registration ... OK
 * checking dependencies in R code ... WARNING
 'library' or 'require' calls not declared from:
   ‘PEcAn.DB’ ‘doParallel’ ‘raster’
 'library' or 'require' calls in package code:
   ‘PEcAn.DB’ ‘doParallel’ ‘raster’ ‘rgdal’
   Please use :: or requireNamespace() instead.
   See section 'Suggested packages' in the 'Writing R Extensions' manual.
-Namespace in Imports field not imported from: ‘stringr’
-  All declared Imports should be used.
+Missing or unexported objects:
+  ‘PEcAn.utils::download_file’ ‘PEcAn.utils::need_packages’
 * checking S3 generic/method consistency ... OK
 * checking replacement functions ... OK
 * checking foreign function calls ... OK
 * checking R code for possible problems ... NOTE
-call_MODIS: no visible global function definition for ‘write.csv’
 download.LandTrendr.AGB: no visible binding for global variable ‘k’
-download.NLCD: no visible global function definition for ‘dbfile.check’
-download.NLCD: no visible global function definition for ‘db.query’
-download.NLCD: no visible global function definition for
-  ‘dbfile.insert’
-download.thredds.AGB : get_data: no visible global function definition
-  for ‘write.csv’
-download.thredds.AGB: no visible global function definition for
-  ‘foreach’
-download.thredds.AGB: no visible global function definition for
-  ‘stopCluster’
-extract_NLCD: no visible global function definition for ‘dbfile.check’
-extract_NLCD: no visible global function definition for ‘db.query’
-extract_NLCD: no visible global function definition for ‘raster’
-extract_NLCD: no visible global function definition for ‘SpatialPoints’
-extract_NLCD: no visible global function definition for ‘CRS’
-extract_NLCD: no visible global function definition for ‘spTransform’
-extract_NLCD: no visible global function definition for ‘crs’
-extract_NLCD: no visible global function definition for ‘extract’
```

Comment:
Look at all those lines removed! 🥇

Reply:
@infotroph I think I messed up a little. data.remote log file wasn't supposed to be here. I might have messed up while rebasing.